### PR TITLE
Request methods module

### PR DESCRIPTION
**Pull request by marcheing**

### CODE REVIEW

File: lib/likeno/base.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
+module Likeno
```
Comment (diegoamc) [inline lib/likeno/base.rb]: This file contains a lot of duplication with [Likeno::Entity](https://github.com/mezuro/likeno/blob/request_methods_module/lib/likeno/entity.rb). Is that right?

Comment (marcheing) [inline lib/likeno/base.rb]: Nope, it definitely isn't :)

**Comment by marcheing** [comment]
Mental note: never write commit messages when the screen is more than 50cm away from the eyes.

**Comment by rafamanzo** [comment]
Nice fixes :)

Don't mix commit with wine :wine_glass: :smile:

**Comment by marcheing** [comment]
It wasn't the :wine_glass: 's fault! He was innocent in this matter, I swear!

**Pull request merged**